.github/workflows/_integration_test.yml

```diff
@@ -57,6 +57,12 @@ jobs:
           GOOGLE_CSE_ID: ${{ secrets.GOOGLE_CSE_ID }}
           EXA_API_KEY: ${{ secrets.EXA_API_KEY }}
           NOMIC_API_KEY: ${{ secrets.NOMIC_API_KEY }}
+          WATSONX_URL: ${{ secrets.WATSONX_URL }}
+          WATSONX_APIKEY: ${{ secrets.WATSONX_APIKEY }}
+          WATSONX_TOKEN: ${{ secrets.WATSONX_TOKEN }}
+          WATSONX_PASSWORD: ${{ secrets.WATSONX_PASSWORD }}
+          WATSONX_USERNAME: ${{ secrets.WATSONX_USERNAME }}
+          WATSONX_INSTANCE_ID: ${{ secrets.WATSONX_INSTANCE_ID }}
```

Comment:
It looks like this represents multiple different auth methods. Would it be possible to only use an api key?

Reply:
I only left the environment variable under api key as you suggested